lib/generate-id.js

```diff
@@ -22,11 +22,41 @@ const findId = async ( model, length ) => {
 }
 
 /**
- * Generates a line id and a shabad id, using the database.
+   * Logs the generated id to the console
+   * @param {Number} count the count of id to generate
+   * @param {String} type shabad, line
+   */
+const logGeneratedID = async function ( count, type ) {
+  let i = 0
+
+  if ( type === 'shabad' ) {
+    for ( i = 0; i < count; i += 1 ) {
+      console.log( `${i + 1}. New Shabad ID: `.bold.magenta + `${await findId( Shabads, 3 )}`.rainbow )
+    }
+  } else if ( type === 'line' ) {
+    for ( i = 0; i < count; i += 1 ) {
+      console.log( `${i + 1}. New Line ID: `.bold.magenta + `${await findId( Lines, 4 )}`.rainbow )
+    }
+  } else { // Default log all types
```

Comment:
I'd say the default behaviour should definitely be to require a type, I can't think of a scenario where you'd want to generate the same number of IDs for multiple types.